### ISSUE TITLE
Job amount should accept only valid value

### ISSUE THREAD

**Issue by senthilgcs** [saraav/AatkalThevai-DEV#16]
Can we disable decimal part and also shall we kept max amount id less than 50K. 

As per govt rule any cash transaction above 50k needs PAN so let us keep within limit

**Comment by saraav** [comment]
Yup. this has been addressed in the current build... This  min and max job fees can be controlled via remote configuration..(currently set from 10 to 49999)
